hazelcast/src/main/java/com/hazelcast/multimap/impl/txn/TxnPrepareOperation.java

```diff
@@ -75,12 +77,18 @@ public int getId() {
     @Override
     protected void writeInternal(ObjectDataOutput out) throws IOException {
         super.writeInternal(out);
-        out.writeLong(ttl);
+        // RU_COMPAT_3_9
+        if (out.getVersion().isUnknownOrLessThan(Versions.V3_10)) {
```

Comment:
LGTM, but @vbekiaris should have the last word on it

Reply:
Clarified via Slack, that the correct method was used.

Reply:
👍